translations/dutch.json

```
@@ -133,63 +133,63 @@
-		"Queued polls:": "",
-		"Refresh": "",
-		"No polls queued.": "",
-		"#${number} in queue": "",
+		"The announcement has ended.": "De mededeling is gedaan.",
+		"Battles do not support announcements.": "Gevechten ondersteunen geen mededelingen.",
+		"You are not allowed to use filtered words in announcements.": "Je mag geen gefilterde woorden gebruiken in mededelingen.",
+		"There is already a poll or announcement in progress in this room.": "Er is al een poll of mededeling bezig in deze room.",
+		"An announcement was started by ${user.name}.": "Een mededeling is gestart door ${user.name}.",
+		"There is no announcement running in this room.": "Er is momenteel geen mededeling in deze room.",
+		"There is no timer to clear.": "Er is geen timer om te verwijderen.",
+		"The announcement timer was turned off.": "De mededeling timer is af gezet.",
+		"Invalid time given.": "Ongeldige tijd opgegeven.",
+		"The announcement timer is off.": "De mededeling timer staat uit.",
+		"The announcement was ended by ${user.name}.": "De mededeling is be&euml;indigd door ${user.name}.",
+		"Accepts the following commands:": "Aanvaardt de volgende commands:",
 
+		"That option is not selected.": "Die optie is niet geselecteerd.",
+		"You have already voted for this poll.": "Je hebt al gestemd op deze poll.",
+		"No options selected.": "Geen, opties geselecteerd.",
+		"you will not be able to vote after viewing results": "na het bekijken van het resultaat kan je niet meer stemmen",
+		"View results": "Bekijk resultaten",
+		"You can't vote after viewing results": "Je kan niet stemmen na het bekijken van de resultaten",
+		"The poll has ended &ndash; scroll down to see the results": "De poll is gedaan &ndash; scroll naar beneden om het resultaat te zien",
+		"Vote for ${num}": "Stem voor ${num}",
+		"Submit your vote": "Breng uw stem uit",
+		"Quiz": "Quiz",
+		"Poll": "Poll",
+		"Submit": "Verzenden",
+		"ended": "be&euml;indigd",
+		"votes": "stemmen",
+		"delete": "verwijder",
+		"Poll too long.": "Poll is te lang.",
+		"Battles do not support polls.": "Gevechten ondersteunen geen polls.",
+		"You are not allowed to use filtered words in polls.": "Je mag geen gefilterde woorden gebruiken in polls.",
+		"Not enough arguments for /poll new.": "Niet genoeg regels/opties voor /poll new.",
+		"Too many options for poll (maximum is 8).": "Te veel opties voor een poll (maximum is 8).",
+		"There are duplicate options in the poll.": "Er zijn dubbele opties in de poll.",
+		"${user.name} queued a poll.": "${user.name} heeft een poll in de wachtrij gezet.",
+		"A poll was started by ${user.name}.": "Een poll was gestart door ${user.name}.",
+		"The queue is already empty.": "De wachtrij is al leeg.",
+		"Cleared poll queue.": "Leeggemaakt.",
+		"Room \"${roomid}\" not found.": "Room \"${roomid}\ is niet gevonden.",
+		"Can't delete poll at slot ${slotString} - \"${slotString}\" is not a number.": "Kan poll op plek ${slotString} niet verwijderen.",
```

```suggestion
		"Can't delete poll at slot ${slotString} - \"${slotString}\" is not a number.": "Kan poll op plek ${slotString} niet verwijderen - \"${slotString}\" is niet een nummer.",
```
Idk if you didn't translated the whole thing intentionally so this is a suggestion.